ui/docs/reference/platform/object/transport-protocols.mdx

```diff
@@ -0,0 +1,29 @@
+---
+order: 3
+inline: true
+title: transport
+---
+# Transport Protocols
+
+## Uploading and Downloading Objects
+
+The Object Service does not directly upload and download objects, but acts as
+an intermediary on behalf of other data storage service providers. To upload an
+object, clients must make a request to the Object Service describing the object
+to upload (`object.uploadObject`), and the Object Service responds with details
+for the client to perform the upload directly.
+
+Similarly, to download an object (`object.downloadObject`), a request must be
+made to the Object Service describing the object, and the Object Server
+responds with information for the client to download the object directly
+(typically a URL).
```

Comment:
This description doesn't seem to match what is built. I would expect something like:

To download an object (`object.downloadObject`), a request must be made to the Object Service with the object name and an ordered list of acceptable protocols (see below), and the Object Server responds with the server-selected protocol, along with the details (typically a URL) for the client to download the object directly.

Reply:
I think this description is good but is very specific to the current implementation choice. Perhaps this would be good addition to the schema docs. The purpose of this page is to give a very high level view but the specifics of the data exchange is defined at a schema reference level. Another advantage of including only our high level summary here is that as the API evolves this high level description will still apply.